pkg/reconciler/pipeline/dag/dag_test.go

```diff
@@ -526,10 +484,6 @@ func TestBuild_InvalidDAG(t *testing.T) {
 		name: "duplicate-tasks",
 		spec: v1beta1.PipelineSpec{Tasks: []v1beta1.PipelineTask{a, a}},
 		err:  "duplicate pipeline task",
-	}, {
-		name: "invalid-task-name-from",
-		spec: v1beta1.PipelineSpec{Tasks: []v1beta1.PipelineTask{invalidTaskFrom}},
```

Comment:
Do we test the result -> param dependency type elsewhere?
We could replace the From dependency with that and keep the test

Reply:
Thanks for the suggestion, replaced.